src/main.js

```diff
@@ -158,6 +159,30 @@ client.on("message", msg => {
     msg.reply("Feel free to submit a pull request!\nhttps://github.com/someguynamedben/D.A.D.Bot");
   }
 
+  // Select a random anime from the anime-list.csv file
+  if (msg.content === '!anime') {
+    const flavorMessages = [
+      '_May God have mercy on your soul._',
+      `_I'm not sure what to say._`,
+      `_${msg.author.username} may have bribed me to pick this one..._`,
+    ]
+    selectRandomAnime().then(anime => {
+      if (anime.isError) {
+        msg.reply(new MessagePayload(msg.channel, {

```

Comment:
MessagePayload here isn't necessary, though in a happier world it would have been nice to do rich formatting (e.g. an image with a link to the anime) - I think that's possible with MessagePayload? Couldn't tell from the API.